### PR TITLE
Site Creation: Duplicate reused layouts

### PR DESCRIPTION
**Pull request by oguzkocer**
When we copied the site creation flow in #8515 to leave the current flow untouched we didn't duplicate the layouts which means if we make a change to the layout it'll change the existing flow as well. I've duplicated the domain layouts as well as the site creation activity layout. I believe these are the only ones we'll use in the new site creation flow, but if I have missed something, this is a good PR to add it.

I've not made any changes to the layouts except for reformatting them. That'll let me reformat the layouts as I work on them without affecting the changeset size.

To test:
* Enable `NEW_SITE_CREATION_ENABLED` flag
* Go to new site creation flow and verify that it's working the same as in `feature/master-site-creation`, specifically the domain screen

P.S: @malinajirka We need to be extra careful about not changing any existing resources for the current site creation flow including the dimensions and strings. I've done a pass at the current changes and I think we are fine, but it's good to keep this in mind as we work on the rest of the changes.

### CODE REVIEW

**Comment by oguzkocer** [comment]
@malinajirka It looks like some of the layouts are not used anymore, so Travis is failing, I'll check the issue in the morning.

**Comment by malinajirka** [comment]
Good point @oguzkocer, thanks for fixing this! I'll make sure to double check it before we merge `feature/master-site-creation` into `develop` ;).

I've locally reverted the changes in `SiteCreationDomainAdapter` and tested both flows and everything works as expected. So if you don't change anything else but `SiteCreationDomainAdapter.java`, feel free to merge it;).

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SiteCreationDomainAdapter.java [hunk 1]
```diff
@@ -140,14 +140,16 @@ void setData(boolean isLoading, String carryOverDomain, String selectedDomain, L
         switch (viewType) {
             case VIEW_TYPE_HEADER:
                 return new HeaderViewHolder(LayoutInflater.from(parent.getContext())
-                                                          .inflate(R.layout.site_creation_domain_header, parent,
+                                                          .inflate(R.layout.new_site_creation_domain_header, parent,
```
[the remark this answers is not in the document]
Comment (oguzkocer) [reply WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SiteCreationDomainAdapter.java]: Yeap, that was a mistake, it's fixed in a82ff95ea94f866bf7e1025615003fc902a1c921.

I know you mentioned that I can merge it, but I like the reviewer merging the PRs except for cases where the timing matters. This one can wait until tomorrow :)

P.S: CircleCI is failing on an unrelated matter and currently it's not a requirement for our review process as far as I can tell from the recent p2 post.

**Comment by oguzkocer** [comment]
@malinajirka Travis seems to be failing in the base branch which was carried over to this branch during merge. If it's OK with you, let's merge the open PRs to the base branch and fix all the issues in one small PR. I don't want to make unrelated changes in the PR.

For the `drawableTint` issue, it should be fixed in the domain UI PR, we can either ignore that, or just remove it temporarily.

**Comment by malinajirka** [comment]
LGTM! Thanks!

**Pull request merged**